src/MockBot.UnitTests/MockBot.UnitTests.csproj

```
@@ -12,12 +12,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Buerokratt.Common" Version="1.0.33" />
+    <PackageReference Include="Buerokratt.Common" Version="1.0.39" />
     <PackageReference Include="FluentAssertions" Version="6.7.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
     <PackageReference Include="MockLogging" Version="0.1.8" />
     <PackageReference Include="Moq" Version="4.18.1" />
     <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" />
+	<PackageReference Include="System.Net.Http" Version="4.3.4" />
+	<PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```

supernit: Rogue spaces
```suggestion
    <PackageReference Include="System.Net.Http" Version="4.3.4" />
    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```

---

Spaces not tabs in here - sheesh :D